.github/workflows/failure_notification.yaml

```diff
@@ -0,0 +1,20 @@
+name: Check for master failure
+on:
+  workflow_run:
+    workflows: [Build and Test]
```

Comment:
Why not the Conda or Analysis builds?

Reply:
Yes, can do.  Just wanted to start with something.